static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -120,6 +121,15 @@ const initialise = (window) => {
     });
 };
 
+const recordFirstPrebidStarted = once(() => {
+    markTime('First Prebid Ad Request Started');
+});
+
+const recordFirstPrebidEnded = once(() => {
+    markTime('First Prebid Ad Request Ended');
+});
+
+
```

Comment:
@ioanna0 Doesn't the system create new instances of prebidAdUnit for every slot?
So this will record an event for the first time these events happen for each slot (so won't trigger when the slot is refreshed). But we want to record it for the first time it happens on the page (regardless of the slot) - and also the first time it happens on top-above-nav

Reply:
It won't create new instances for each slot as there is no reference to the slot. This will already only be loaded once - with the use of lodash's `once`. Once for each page regardless of the slot, without refreshes. First time this happens for `top-above-nav` is captured below on the (new line) 166